README.md

```diff
@@ -67,6 +67,47 @@ provided packages and install only the desired slices into the *myrootfs*
 folder, according to the slice definitions available in the
 ["ubuntu-22.04" chisel-releases branch](<https://github.com/canonical/chisel-releases/tree/ubuntu-22.04>).
 
+## Support for Pro archives
+
+To fetch and install packages from Ubuntu Pro archives,
+the archive has to be defined with the `archives.<name>.pro` field in
+`chisel.yaml`:
+
+
+```yaml
+# chisel.yaml
+format: v1
+archives:
+  <name>:
+    pro: <value>
+    ...
+...
+```
+
+The following Pro archives are currently supported:
+
+| `pro` value | Archive URL | Related Ubuntu Pro service |
+| - | - | - |
+| fips         | https://esm.ubuntu.com/fips/ubuntu         | fips         |
+| fips-updates | https://esm.ubuntu.com/fips-updates/ubuntu | fips-updates |
+| apps         | https://esm.ubuntu.com/apps/ubuntu         | esm-apps     |
+| infra        | https://esm.ubuntu.com/infra/ubuntu        | esm-infra    |
+
+If the system is using the [Pro client](https://ubuntu.com/pro/tutorial), and the
+services are enabled, the credentials will be automatically picked up from
+`/etc/apt/auth.conf.d/`. However, the default permissions of the credentials need
+to be changed for Chisel to be able to read them. Example workflow:
+```shell
+sudo pro enable fips
+
```

Comment:
In the discussion I had with Gustavo today we mentioned explicitly adding a "complete example" so that the users know what do we mean by each of the terms and they can see the "whole thing".

Reply:
Ok. I'd then choose a different service because the complete example with `fips` requires a reboot of the machine for `fips` to be properly enabled (not that it matters for Chisel, but still)